lib/python/qmk/keyboard.py

```diff
@@ -72,10 +91,11 @@ def rules_mk(keyboard):
     return rules
 
 
-def render_layout(layout_data, key_labels=None):
+def render_layout(layout_data, render_ascii, key_labels=None):
     """Renders a single layout.
     """
     textpad = [array('u', ' ' * 200) for x in range(50)]
+    style = 'ascii' if render_ascii else 'unicode'
```

Comment:
This will make the lines below more readable.

```suggestion
    style = 'ascii' if render_ascii else 'unicode'
    box_chars = BOX_DRAWING_CHARECTERS[style]
```